app/javascript/src/modules/virtex_3d_viewer.js

```diff
@@ -1,47 +1,17 @@
 'use strict';
 
```

Comment:
Lots of the code below can go away since the model-viewer component declaratively sets up the viewer. It's hard to see from the diff but I also used JavaScript directly to wire up the zooming buttons, rather than relying on jQuery.